features/contenteditable.yml

```
@@ -0,0 +1,9 @@
+name: Contenteditable
+description: The `contenteditable` attribute represents whether the element should be editable by the user. If so, the browser modifies its widget to allow editing.
```

I'm not sure I understand what "the browser modifies its widget to allow editing" means. I see this comes from MDN, but I think it could be improved.

```suggestion
description: The `contenteditable` attribute allows the user to edit the content of an element, for example by inserting or deleting text.
```